DuckDuckGo/AppDelegate.swift

```diff
@@ -42,18 +50,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         testing = ProcessInfo().arguments.contains("testing")
         if testing {
             window?.rootViewController = UIStoryboard.init(name: "LaunchScreen", bundle: nil).instantiateInitialViewController()
+            return true
```

Comment:
No need to progress beyond this point for unit tests. This should have been done when this was added.